lib/client-api/src/config_api.ts

```diff
@@ -14,10 +48,17 @@ export default class ConfigApi {
 
   _renderMain() {
     // do initial render of story
+    debugger;
```

Comment:
hmm not sure it should be still here 😉

Reply:
it should not, i am actually battling with node-gyp and cannot build storybook anymore, will be updating PR HOPEFULLY tonight; post reformating